Framework/src/PostProcessingRunner.cxx

```diff
@@ -82,16 +91,14 @@ void PostProcessingRunner::init(const PostProcessingRunnerConfig& runnerConfig,
   }
 
   // configuration of the database
-  mDatabase = DatabaseFactory::create(mRunnerConfig.database.at("implementation"));
-  mDatabase->connect(mRunnerConfig.database);
-  ILOG(Info, Support) << "Database that is going to be used > Implementation : " << mRunnerConfig.database.at("implementation") << " / "
-                      << " Host : " << mRunnerConfig.database.at("host") << ENDM;
+  mSourceDatabase = configureDatabase(mRunnerConfig.sourceDatabase, "Source");
+  mDestinationDatabase = configureDatabase(mRunnerConfig.destinationDatabase, "Destination");
 
   mObjectManager = std::make_shared<ObjectsManager>(mTaskConfig.taskName, mTaskConfig.className, mTaskConfig.detectorName, mRunnerConfig.consulUrl);
   mObjectManager->setActivity(mActivity);
-  mServices.registerService<DatabaseInterface>(mDatabase.get());
+  mServices.registerService<DatabaseInterface>(mSourceDatabase.get());
   if (mPublicationCallback == nullptr) {
-    mPublicationCallback = publishToRepository(*mDatabase);
+    mPublicationCallback = publishToRepository(*mDestinationDatabase);
```

Comment:
If done like this, it will only work in cases when a postprocessing task is ran with `o2-qc-run-postprocessing`, that is without DPL. When we run with DPL, this callback is set by `PostProcessingDevice` to publish objects into DPL, then CheckRunners receive them, apply Checks and store them to the database they use.

I think it will be very misleading if it works only for some cases.

How about getting rid of the `destinationRepo` parameter and using always the main repo as the destination?

Reply:
yes, we can do that for the destinatino repo. I did not think of this issue and thought that why not add both source and destination. I'll amend the PR.

Reply:
I have pushed a commit